src/modules/authentication/components/Login/login.css

```diff
@@ -0,0 +1,76 @@
+.Auth-container{
+    background-image: url(../../../../assets/images/bg1.png);
+    background-position: center center;
+    background-size: cover;
+}
+.parent-container{
+    flex-direction: column;
+}
+.login-container{
+    width: 40%;
+    background: rgba(49, 89, 81, 0.9);
```

Comment:
In this project, we will need to implement dark/light mode feature. Using fixed colors is not reccomended as it will make this near impossible. We will need too classes for one for dark, light mode that contain the corresponding variables for each theme and then re-use these colors as needed.

Reply:
done